src/cms/preview-templates/HomePagePreview.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { marked } from "marked"
+import HomePage from '../../templates/home-page'
+
+const HomePagePreview = ({ entry }) => {
+  const { tiles } = entry.getIn(['data']).toJS()
+
+  tiles.forEach(tile => {
+    if (tile.body) tile.body = marked.parse(tile.body)
+  })
+
+  return <HomePage tiles={tiles} />
```

Comment:
This is a good start for the preview. My hope is that we can actually load the App Shell and render the component directly inside that. This will give it any CSS or React Context it might need to render the preview faithfully to how it'll appear in Network360.

Reply:
Example: 
```ts
import Shell from "host/Shell"

<Shell renderApp={() => <HomePage tiles={tiles} />} />
```

Reply:
I'll see if I can figure out rendering within the app shell. :+1: